contracts/interfaces/sablier/ISablierV2LockupLinear.sol

```diff
@@ -1,10 +1,113 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Is it just inferring from the Sablier contract?

Reply:
Yeah, the original `ISablierV2LockupLinear.sol` interface in this codebase was just some bullshit that I had created, apparently. The new code in this interface is a direct copy-paste from the official Sablier repo.